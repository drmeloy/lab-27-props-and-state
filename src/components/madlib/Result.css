.Result {
  display: flex;
  flex-direction: column;
  align-items: center;

  p {
    width: 75%;
    text-align: center;
    margin: 50px auto;
    font-size: 2rem;
  }

  button {
    margin-top: 10px;
    height: 38px;
    border-radius: 25px;
    width: 80px;
  }
}



