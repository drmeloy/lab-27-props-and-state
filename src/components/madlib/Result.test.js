import React from 'react';
import { shallow } from 'enzyme';
import Result from './Result';

describe('Result component', () => {
  it('renders Result', () => {
    const wrapper = shallow(<Result words={['hello', 'hi', 'hey']} />);
    expect(wrapper).toMatchSnapshot();
  });
});