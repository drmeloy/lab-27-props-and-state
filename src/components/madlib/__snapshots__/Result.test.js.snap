// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result component renders Result 1`] = `
<div
  className="Result"
>
  <p>
    Today I went to the zoo. I saw a(n)
hello
hi jumping up and down in its tree.
He hey undefined
through the large tunnel that led to its undefined
undefined. I got some peanuts and passed
them through the cage to a gigantic gray undefined
towering above my head. Feeding that animal made
me hungry. I went to get a undefined scoop
of ice cream. It filled my stomach. Afterwards I had to
undefined undefined to catch our bus.
When I got home I undefined my
mom for a undefined day at the zoo.

  </p>
  <button>
    Start over
  </button>
</div>
`;
